assets/styles/components/specials/_pullquotes.scss

```diff
@@ -44,7 +44,8 @@ aside,
 
 .pullquote {
   @extend %pullquote;
-  margin: $pullquote-margin-top 0 $pullquote-margin-bottom;
+  margin: $pullquote-margin-top auto $pullquote-margin-bottom auto;
```

Comment:
You should use the shorthand version:

```diff
-  margin: $pullquote-margin-top auto $pullquote-margin-bottom auto;
+  margin: $pullquote-margin-top auto $pullquote-margin-bottom;
```

Also it would probably be a good idea to use `if-map-get()` on the margin values. See subsequent comment.